concent_api/core/tests/utils.py

```diff
@@ -167,7 +168,7 @@ def _get_deserialized_report_computed_task(
 
     def _get_deserialized_task_to_compute(
         self,
-        timestamp: Optional[str] = None,
+        timestamp: Optional[Union[str, datetime.date]] = None,
```

Comment:
isn't it `datetime.datetime`?, besides, I think `Union[str, datetime.datetime, None]` would be more readable ;)

Reply:
done